charts/pega/README.md

```diff
@@ -153,6 +153,8 @@ Specify the location for the Pega Docker image.  This image is available on Dock
 
 When using a private registry that requires a username and password, specify them using the `docker.registry.username` and `docker.registry.password` parameters.
 
+To avoid specifying the docker registry credentials in values.yaml, create secrets for docker registry credentials. Refer to [Kubernetes secrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) to create docker registry secrets. Specify secret names as a list of comma-separated strings using `docker.imagePullSecretNames` parameter. Kubernetes checks each registry secret of image pull secrets to pull an image from repository, If the specified image is available in one of the provided secrets, kubernetes pull it from that repository. To create docker registry secrets from external secrets, refer [this section](#optional-support-for-providing-credentialscertificates-using-external-secrets-operator).
```

Comment:
a few suggestions, including consistently referencing Docker images with a capital D ;-)>>
To avoid specifying Docker registry credentials in values.yaml, create secrets for docker registry credentials to avoid  exposing these credentials. Refer to [Kubernetes secrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) to create Docker registry secrets. Specify secret names as a list of comma-separated strings using the `docker.imagePullSecretNames` parameter. Kubernetes checks each registry secret of image pull secrets to pull an image from the repository. If the specified image is available in one of the provided secrets, kubernetes will pull it from that repository. To create docker registry secrets from external secrets, refer to [this section](#optional-support-for-providing-credentialscertificates-using-external-secrets-operator).